stix_shifter_modules/alertflex/configuration/alertflex_config.json

```diff
@@ -0,0 +1,55 @@
+{
+    "connection": {
+        "type": {
+            "default": "alertflex"
+        },
+        "host": {
+            "type": "text"
+        },
+        "port": {
+            "default": 8181,
+            "type": "number"
+        },
+        "help": {
+            "default": "https://github.com/olegzhr/cnode/blob/master/doc/integration.rst",
+            "type": "link"
+        },
+        "limit": {
+            "default": 1000,
+            "max": 10000,
+            "type": "number"
+        },
+        "timeout": {
+            "default": 1,
+            "max": 60,
+            "type": "number"
+        },
+        "concurrent": {
+            "default": 4,
+            "max": 60,
+            "type": "number"
+        },
+        "cert": {
+            "type": "password",
+            "optional": true
+        },
+        "sni": {
+            "type": "text",
+            "optional": true
+        },
+        "selfSignedCert": {
+            "type": "password",
+            "optional": true
+        }
+    },
+    "configuration": {
+        "auth": {
+            "username": {
+                "type": "password"
+            },
+            "password": {
+                "type": "password"
+            }
+        }
+    }
+}
```

Comment:
we have made few changes in the base config files and also its naming conventions. please name this file `config.json`.  you can have a look at dummy or qradar module config files: https://github.com/opencybersecurityalliance/stix-shifter/tree/develop/stix_shifter_modules/async_dummy/configuration